README.md

```diff
@@ -199,163 +152,7 @@ Finally, Fantomas has the same Greek root as "[phantom](https://en.wiktionary.or
-there is a `[Feature]Test.fs` file. Examples:
- - *StringTests.fs*
- - *UnionTests.fs*
- - ...
-
-Most of the tests are really simple and have this simple algorithm:
-_assert that **format \[F# CODE\]** is equal to **\[FORMATTED F# CODE\]**_.
-
-Example (from *UnionTests.fs*):
-```fsharp
-[<Test>]
-let ``discriminated unions declaration``() =
-    formatSourceString false "type X = private | A of AParameters | B" config
-    |> prepend newline
-    |> should equal """
-type X =
-    private
-    | A of AParameters
-    | B
-"""
-```
-
-#### The `CodePrinter.genParsedInput` function: rewrites formatted code
-`CodePrinter.genParsedInput` (see [CodePrinter.fs](src/Fantomas/CodePrinter.fs)): what it
-basically does is traversing the AST corresponding to the code to format, and
-rewriting it according to the provided formatting options.
-
-#### The `FormatConfig` type: format settings
-Settings such as :
- - indent values in spaces
- - maximum page width
- - ...
-
-See [CodePrinter.fs](src/Fantomas/CodePrinter.fs).
-
-### How to play with Fantomas on F# Interactive
-The [CodeFormatter.fsx](src/Fantomas/CodeFormatter.fsx) script file
-allows you to test the code formatting behavior. See the function `formatSrc: string -> unit`
-that formats the string in input and prints it.
-
-### Video series
-
-There is a [YouTube video series](https://www.youtube.com/playlist?list=PLvw_J2kfZCX3Mf6tEbIPZXbzJOD1VGl4K) on how Fantomas internally works.
-
-### Ionide
-
-When you want to contribute to this project in VSCode with Ionide, there  is a trick you need to know to debug Unit tests.
-
-After checking out the repository, open a terminal and set the `VSTEST_HOST_DEBUG` environment variable to `1`.
-
-In PowerShell:
-
-> $env:VSTEST_HOST_DEBUG=1
-
-or in Bash:
-
-> VSTEST_HOST_DEBUG=1
-
-Run a single unit test with `dotnet test --filter`.
-
-> cd .\src\Fantomas.Tests\
-> dotnet test --filter "record declaration"
-
-The output looks like:
-
-```
-Test run for C:\Temp\fantomas\src\Fantomas.Tests\bin\Debug\netcoreapp3.1\Fantomas.Tests.dll(.NETCoreApp,Version=v3.1)
-Microsoft (R) Test Execution Command Line Tool Version 16.3.0
-Copyright (c) Microsoft Corporation.  All rights reserved.
-
-Starting test execution, please wait...
-
-A total of 1 test files matched the specified pattern.
-Host debugging is enabled. Please attach debugger to testhost process to continue.
-Process Id: 20312, Name: dotnet
-```
-
-And we can now attach to the unit testing process.
-
-![Run .NET Core Attach ](./docs/fantomas-debug-vscode-1.png)
-
-![Choose process id](./docs/fantomas-debug-vscode-2.png)
-
-**Press the play button once the process has been chosen!**
-This might be a bit strange but you need to press play in order for the debugger to start working.
-
-![Hit the breakpoint](./docs/fantomas-debug-vscode-3.png)
-
-Check out this [video fragment](https://youtu.be/axHIazqiO9E?t=65) to see this in action.
+See [Contribution guide](./CONTRIBUTING.md).
```

Comment:
You've called it "the contribution guidelines" in most other places, but let's make that capitalization on Contribution and Guide consistent.  Either upper or lower works for me.  You probably need a "the" before it, too.